### PR TITLE
chore: cidoc unit tests for different document types

### PR DESCRIPTION
**Pull request by H-Fin**
#### What do these changes do/fix?
Tests that interactions work as expected. Adds unit tests for all possible iterations of the side panel layouts and section headers that are dependent on the document type.

https://zenhub.ibm.com/app/workspaces/discovery-unified-5728bf39d2c20f776e05d0ef/issues/watson-discovery/docviz-squad-issue-tracker/236
#### How do you test/verify these changes?
run `yarn test: watch CIDocument.spec` and see that tests pass
#### Have you documented your changes (if necessary)?
No
#### Are there any breaking changes included in this pull request?
N/A

### CODE REVIEW

File: packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx [hunk 1]
```diff
@@ -48,22 +56,36 @@ describe('<CIDocument />', () => {
...
-        const tabs = getByTestId('document-info-tabs');
-        return globalGetByText(tabs, 'Relations');
-      });
-      fireEvent.click(tabButton);
+    it('loads the correct tabs and pane sections titles', async () => {
+      // Attribute and Relations tabs exist
+      const attributesTab = getByTestId('attributes-tab');
+      const relationsTab = getByTestId('relations-tab');
+      expect(attributesTab.textContent).toEqual('Attributes');
+      expect(relationsTab.textContent).toEqual('Relations');
```
Comment (jhpedemonte) [inline packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx]: How hard is it to also add a test that filters and metadata tabs _do not_ exist?

File: packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx [hunk 2]
```diff
@@ -160,6 +182,42 @@ describe('<CIDocument />', () => {
       globalGetByText(filters, 'Intellectual Property');
     });
 
+    it('loads the correct tabs and pane sections titles', async () => {
+      // Filters and Metadata tabs exist
+      const filterTab = getByTestId('filters-tab');
+      const metadataTab = getByTestId('metadata-tab');
+      expect(filterTab.textContent).toEqual('Filters');
+      expect(metadataTab.textContent).toEqual('Metadata');
```
Comment (jhpedemonte) [inline packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx]: And here: test that attributes and relations tabs _do not_ exist

File: packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx [hunk 3]
```diff
@@ -160,6 +182,42 @@ describe('<CIDocument />', () => {
...
+      const detailsAttributes = getByTestId('details-pane-attributes');
+      expect(detailsCategories.textContent).toContain('Intellectual Property');
+      expect(detailsTypes.textContent).toContain('Nature: Definition, Party: None');
+      expect(detailsAttributes.textContent).toContain('DefinedTerm (1)');
+    });
+
+    it('displays the correct party information on the details pane', async () => {
+      // Expects metadata pane to be hidden until selected
+      const metadataPane = getByTestId('metadata-pane').parentElement!;
+      expect(metadataPane.getAttribute('aria-hidden')).toEqual('true');
```
Comment (jhpedemonte) [inline packages/discovery-react-components/src/components/CIDocument/components/CIDocument/__tests__/CIDocument.spec.tsx]: Neat!

**Comment by H-Fin** [comment]
@jhpedemonte Changes have been made!

**Pull request merged**